shared/src/persistence/elasticsearch/workitems/getDocumentQCInProgressForSection.js

```diff
@@ -0,0 +1,69 @@
+const { search } = require('../searchClient');
+
+exports.getDocumentQCInProgressForSection = async ({
+  applicationContext,
+  judgeUserName,
+  section,
+}) => {
+  console.log('!!!', section);
+  const query = {
+    body: {
+      query: {
+        bool: {
+          must: [
+            {
+              prefix: { 'pk.S': 'work-item|' },
+            },
+            {
+              prefix: { 'sk.S': 'work-item|' },
+            },
+            {
+              term: {
+                'section.S': section,
+              },
+            },
+            {
+              term: {
+                'caseIsInProgress.BOOL': true,
```

Comment:
Is this going to cause problems for the docket section in progress box? Their work items won't necessarily be for cases in progress, will they? Or is this method only being used for petitions?